internal/host/hostcommands/install_cmd.go

```diff
@@ -256,17 +256,27 @@ func (i *installCmd) getDisksToFormat(ctx context.Context, host *models.Host, in
 func constructHostInstallerArgs(cluster *common.Cluster, host *models.Host, inventory *models.Inventory, infraEnv *common.InfraEnv, log logrus.FieldLogger) (string, error) {
 
 	var installerArgs []string
+	var err error
+	hasStaticNetwork := (infraEnv != nil && infraEnv.StaticNetworkConfig != "") || cluster.StaticNetworkConfigured
 
 	if host.InstallerArgs != "" {
-		err := json.Unmarshal([]byte(host.InstallerArgs), &installerArgs)
+		err = json.Unmarshal([]byte(host.InstallerArgs), &installerArgs)
 		if err != nil {
 			return "", err
 		}
 	}
 
-	installerArgs, err := appendDHCPArgs(cluster, host, inventory, installerArgs, log)
-	if err != nil {
-		return "", err
+	if !hasStaticNetwork {
```

Comment:
Can/should we do something more clever here?

I remember people mentioning that there are many cases where users configure static networking only for a subset of the interfaces on a host.

Should we only be omitting the ones they have specifically set static networking for?